R/run-url.R

```diff
@@ -112,7 +112,7 @@ runGist <- function(gist, destdir = NULL, ...) {
     stop('Unrecognized gist identifier format')
   }
 
-  runUrl(gistUrl, filetype = ".tar.gz", destdir = destdir, ...)
+  runUrl(gistUrl, filetype = ".zip", destdir = destdir, ...)
```

Comment:
The download URL that's embedded in the gist UI has a `.zip` extension. If you replace that with `.tar.gz`, you can get a tarball. Scraping the HTML doesn't feel great and I don't see that GitHub documents either of these direct-download approaches.

An alternate approach would be to look at the `files.FILENAME.raw_url` within a gist API response. Would that be less fragile?
https://developer.github.com/v3/gists/#response